lms/djangoapps/discussion_api/api.py

```diff
@@ -487,7 +488,7 @@ def _check_editable_fields(cc_content, data, context):
     )
 
 
-def _do_extra_actions(api_content, cc_content, request_fields, actions_form, context):
+def _do_extra_actions(api_content, cc_content, request_fields, actions_form, context, request):
```

Comment:
@jcdyer i wonder why the assert is there or why should we put any keyerror while this function only does extra actions required after a thread/comment is created or updated and all the field validations are already made before entring to this function. So i would like to remove that on whole. Does it make sense?